lib/samples/find_route/find_route_sample.dart

```diff
@@ -143,6 +157,7 @@ class _FindRouteSampleState extends State<FindRouteSample> {
   }
 
   Future<void> initRouteParameters() async {
+    // create default route parameters.
     final parameters = await _routeTask.createDefaultParameters();
```

Comment:
this could all be combined with `..` operators. Plus it could just assign to `_routeParameters` directly -- no need to a temporary `parameters`.